### DISCUSSION TITLE
Github Codespaces don't correctly register upstream when opening a template repository

### DISCUSSION THREAD

**Discussion by RockBacon9922** [community/community#62474] [Codespaces]
### Select Topic Area

Bug

### Body

In addition to issue #61200

I have forked a template repository.
I have tried on the repo main view to create a codespace on one of the branches. Because it is a template the button for edit code is replaced with use template.
Therefore I try creating a codespace from github.com/codespaces. It creates the repo but doesn't register the upstream origin/remote.

**Comment by shcheklein** [comment]
Hitting the same issue with https://github.com/shcheklein/ultralytics. It doesn't pick up any branches and it seems git is not initialized at all.

**Reply by shcheklein** [reply]
For now I've "solved it" by turning off the template settings for this fork.
